### PR TITLE
Add Tailwind Made

### PR DESCRIPTION
**Pull request by vivgui**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -207,6 +207,7 @@
 - 🎨 [Seminyak Hugo Theme](https://git.habd.as/jhabdas/seminyak) - Hugo theme using Tailwind CSS.
 - 🎨 [Create React App template with Tailwind CSS + TypeScript](https://github.com/dance2die/cra-template-tailwindcss-typescript) - CRA template with support for Tailwind CSS and TypeScript.
 - 🎨 [Windmill Dashboard](https://windmill-dashboard.vercel.app/) - Multi theme, completely accessible dashboard.
+- 🎨 [Tailwind Made](https://tailwindmade.com/) - Paid, developer-friendly templates made with TailwindCSS
```
Comment (innocenzi) [inline readme.md:210]:
```suggestion
- 🎨 [Tailwind Made](https://tailwindmade.com/) - Paid, developer-friendly templates made with Tailwind CSS.
```

**Review by innocenzi — CHANGES_REQUESTED** [review]
Hey, happy to merge after this small change! :)

**Comment by vivgui** [comment]
@innocenzi done 😄

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
Thank you!

**Comment by joshmanders** [comment]
Yay @vivgui!

**Pull request merged**